x-pack/plugins/data_visualizer/public/application/file_data_visualizer/components/file_data_visualizer_view/file_data_visualizer_view.js

```diff
@@ -102,17 +102,26 @@ export class FileDataVisualizerView extends Component {
   };
 
   async loadFile(file) {
-    if (file.size <= this.maxFileUploadBytes) {
+    this.fileSizeChecker = new FileSizeChecker(this.props.fileUpload, file);
+    if (this.fileSizeChecker.check()) {
       try {
         const { data, fileContents } = await readFile(file);
-        this.setState({
-          data,
-          fileContents,
-          fileName: file.name,
-          fileSize: file.size,
-        });
-
-        await this.analyzeFile(fileContents);
+        console.log(file.type);
```

Comment:
This console.log left in on purpose?

Reply:
Removed in https://github.com/elastic/kibana/pull/186956/commits/1b92419b252a87164db31f8a7dcc8bd6ca8211d9